test/module/shared_model/validators/validators_fixture.hpp

```diff
@@ -117,28 +120,51 @@ class ValidatorsTest : public ::testing::Test {
 
  protected:
   void SetUp() override {
-    valid_created_time = iroha::time::now();
+    // Fill fields with valid values
+    created_time = iroha::time::now();
+    public_key_size = 32;
+    hash_size = 32;
+    counter = 1048576;
+    account_id = "account@domain";
+    asset_name = "asset";
+    asset_id = "asset#domain";
+    address_localhost = "localhost:65535";
+    address_ipv4 = "192.168.255.1:8080";
+    address_hostname = "google.ru:8080";
+    role_name = "user";
+    account_name = "admin";
+    domain_id = "ru";
+    detail_key = "key";
+    public_key = std::string(public_key_size, '0');
+    hash = std::string(public_key_size, '0');
+    role_permission = iroha::protocol::RolePermission::can_append_role;
+    grantable_permission =
+        iroha::protocol::GrantablePermission::can_add_my_signatory;
+    quorum = 2;
+    precision = 42;
```

Comment:
Is it fine that precision doesn't match with the one at the `ValidatorsTest::ValidatorsTest()`?